devel/nodejs14/Portfile

```diff
@@ -86,10 +83,36 @@ post-patch {
 
 # use the system libuv instead of the bundled version, as it is fixed for older systems
 if { ${os.platform} eq "darwin" && ${os.major} < 11 } {
+    # on macOS nodejs only builds against libc++
+    # this force is OK as node does not link against any other c++ libs
+    # per @kencu, libcxx is only for 10.5 and 10.6 as a prerequisite for an even-newer libcxx (macports-libcxx)
+    depends_lib-append      port:libcxx
     depends_lib-append    path:lib/libuv.dylib:libuv
     configure.args-append --shared-libuv
 }
 
+#apparently even macos14 needs the flags that i originally included for older macs.
+#so i am going to make the 'ignore warnings as errors' applicable for all targets
+configure.cppflags-append   -Wno-incompatible-function-pointer-types -Wno-enum-constexpr-conversion
```

Comment:
Firstly, these are clangisms, AFAIR, so please do not use them unconditionally. GCC may need similar flags, but of its own.
Secondly, `-Wno-error=` is preferable, since it retains a warning.

Reply:
how do you protect flags in a Portfile so that it's for clang and not gcc?